src/main/java/oss/fosslight/api/controller/v1/ApiProjectController.java

```diff
@@ -478,6 +478,39 @@ public ResponseEntity<FileSystemResource> getPrjBomExport(
 			return null;
 		}
 	}
+
+	@ApiOperation(value = "Project Bom Tab Export Json", notes = "Project > Bom tab Export Json")
+	@ApiImplicitParams({
+			@ApiImplicitParam(name = "_token", value = "token", required = true, dataType = "String", paramType = "header")
+	})
+	@GetMapping(value = {API.FOSSLIGHT_API_PROJECT_BOM_EXPORT_JSON})
+	public CommonResult getPrjBomExportJson(
+			@RequestHeader String _token,
+			@ApiParam(value = "Project id", required = true) @RequestParam(required = true) String prjId) {
+
+		T2Users userInfo = userService.checkApiUserAuth(_token);
+		Map<String, Object> resultMap = new HashMap<String, Object>();
+
+		try {
+			Map<String, Object> paramMap = new HashMap<String, Object>();
+			List<String> prjIdList = new ArrayList<String>();
+			prjIdList.add(prjId);
+			paramMap.put("userId", userInfo.getUserId());
+			paramMap.put("userRole", userRole(userInfo));
+			paramMap.put("prjId", prjIdList);
+			paramMap.put("distributionType", "normal");
```

Comment:
How about changing it like below? It would be more readable.

```java
List<String> prjIdList = new ArrayList<String>();
prjIdList.add(prjId);

Map<String, Object> paramMap = new HashMap<String, Object>();
paramMap.put("userId", userInfo.getUserId());
paramMap.put("userRole", userRole(userInfo));
paramMap.put("prjId", prjIdList);
paramMap.put("distributionType", "normal");
```